backend/app/legacy/core/proposals.py

```diff
@@ -10,6 +10,10 @@ def get_proposals_addresses(epoch: Optional[int]) -> List[str]:
     return proposals.get_proposal_addresses(epoch)
 
 
+def get_proposals_cid() -> str:
```

Comment:
We shouldn't add any more code to `legacy` package. Pls, call a contract function directly in the service and remove this one